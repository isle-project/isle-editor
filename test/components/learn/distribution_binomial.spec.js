// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import LearnBinomialDistribution from '@isle-project/components/learn/distribution-binomial/main.js';


// TESTS //

describe( '<LearnBinomialDistribution />', function test() {
	it( 'renders a learning component', () => {
		const learning = renderer.create( <LearnBinomialDistribution /> );
		expect( learning ).toMatchSnapshot();
	});
});
