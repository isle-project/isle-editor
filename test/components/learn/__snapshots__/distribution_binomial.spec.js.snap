// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LearnBinomialDistribution /> renders a learning component 1`] = `
<div
  className="card"
  style={
    Object {
      "margin": "10px auto",
      "maxWidth": 1200,
    }
  }
>
  <h3
    className="card-header"
  >
    binomial-distribution
  </h3>
  <div
    className="card-body"
  >
    <ul
      className="nav nav-tabs"
      onKeyDown={[Function]}
      role="tablist"
    >
      <li
        className="nav-item"
        role="presentation"
      >
        <button
          aria-controls="binomial-tabs-tabpane-0"
          aria-selected={true}
          className="nav-link active"
          data-rr-ui-event-key="0"
          disabled={false}
          id="binomial-tabs-tab-0"
          onClick={[Function]}
          role="tab"
          type="button"
        >
          <div />
        </button>
      </li>
      <li
        className="nav-item"
        role="presentation"
      >
        <button
          aria-controls="binomial-tabs-tabpane-1"
          className="nav-link"
          data-rr-ui-event-key="1"
          disabled={false}
          id="binomial-tabs-tab-1"
          onClick={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <div />
        </button>
      </li>
      <li
        className="nav-item"
        role="presentation"
      >
        <button
          aria-controls="binomial-tabs-tabpane-2"
          className="nav-link"
          data-rr-ui-event-key="2"
          disabled={false}
          id="binomial-tabs-tab-2"
          onClick={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <div />
        </button>
      </li>
      <li
        className="nav-item"
        role="presentation"
      >
        <button
          aria-controls="binomial-tabs-tabpane-3"
          className="nav-link"
          data-rr-ui-event-key="3"
          disabled={false}
          id="binomial-tabs-tab-3"
          onClick={[Function]}
          role="tab"
          tabIndex={-1}
          type="button"
        >
          <div />
        </button>
      </li>
    </ul>
    <div
      className="tab-content"
    >
      <div
        aria-labelledby="binomial-tabs-tab-0"
        className="tab-pane active"
        id="binomial-tabs-tabpane-0"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-md-5"
            />
            <div
              className="col-md-7"
            />
          </div>
        </div>
      </div>
      <div
        aria-labelledby="binomial-tabs-tab-1"
        className="tab-pane"
        id="binomial-tabs-tabpane-1"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-md-4"
            />
            <div
              className="col-md-8"
            />
          </div>
        </div>
      </div>
      <div
        aria-labelledby="binomial-tabs-tab-2"
        className="tab-pane"
        id="binomial-tabs-tabpane-2"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-md-4"
            />
            <div
              className="col-md-8"
            />
          </div>
        </div>
      </div>
      <div
        aria-labelledby="binomial-tabs-tab-3"
        className="tab-pane"
        id="binomial-tabs-tabpane-3"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col-md-4"
            />
            <div
              className="col-md-8"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
